lib/mrb/scripts/command_line/grndb.rb

```diff
@@ -402,6 +405,37 @@ def failed_to_open(name)
           failed(message)
         end
       end
+
+      class Recoverer
+        attr_writer :database
+
+        def initialize
+          @context = Context.instance
+        end
+
+        def recover(force_truncate)
+          @database.recover
+          if force_truncate
+            @database.each do |object|
+              if object.corrupt?
+                logger = @context.logger
+                Dir.foreach(File.dirname(object.path)) do |path|
+                  if path =~ /^#{File.basename(object.path)}.+/
+                    begin
+                      File.unlink("#{File.dirname(object.path)}/#{path}")
+                      message = "Removed file which is related to corrupted #{object.path}: #{path}"
+                      logger.log(Logger::Level::INFO.to_i, __FILE__, __LINE__, "recover", message)
```

Comment:
I think that logging actually removed file to log file is important in this context because standard output may be lost by user without redirecting, is it good enough to use standard output?